internal/action/generate.go

```diff
@@ -239,15 +239,29 @@ func (s *Action) generatePassword(ctx context.Context, c *cli.Context, length, n
 
 	switch generator {
 	case "memorable":
+		isStrict := false
 		if c.Bool("strict") {
+			isStrict = true
+		} else if cfg.IsSet("generate.strict") {
+			isStrict = cfg.GetBool("generate.strict")
+		}
+
+		if isStrict {
```

Comment:
Can you move the isStrict evaluation to a method and just call that here and below? So we have less duplication.